src/sort_cards/sort_cards.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+
+
+def sort_cards(cards):
+    """Sort shuffled list of cards, sorted by rank.
+
+    >>> sort_cards(['3', '9', 'A', '5', 'T', '8', '2', '4', 'Q', '7', 'J', '6', 'K'])
+    ['A', '2', '3', '4', '5', '6', '7', '8', '9', 'T', 'J', 'Q', 'K']
+
+    """
+
+    new_list = []
+    for i, val in enumerate(cards):
+        if val == 'A':
+            new_val = 1
+        elif val == 'T':
+            new_val = 10
+        elif val == 'J':
+            new_val = 11
+        elif val == 'Q':
+            new_val = 12
+        elif val == 'K':
+            new_val = 13
+        else:
+            new_val = int(val)
+        (new_list).append(new_val)
```

Comment:
If these "cards" were carrying any other type of information with them (like say card suit, or even which deck they came from), that would all be lost with this type of operation.